packages/node/src/workers/local-handlers.ts

```diff
@@ -16,7 +16,7 @@ export interface CallApiArgs {
 }
 
 function loadConfig() {
-  const rawConfig = JSON.parse(fs.readFileSync('./config.json', 'utf8'))[0];
+  const rawConfig = JSON.parse(fs.readFileSync('./config.json', 'utf8'));
```

Comment:
So this was actually broken as it wouldn't have worked with more than config? 😄

Reply:
:smile: Kinda. It wasn't broken just not finished.